models/courier/Courier.model.js

```diff
@@ -0,0 +1,49 @@
+import { Schema } from 'mongoose';
+import { addressSchema } from '../helpers/validation';
+import { vehicleType } from '../../constants/vehicleType';
+import { phoneNumberPattern } from '../../helpers/validation';
+import { accountStatus } from '../../constants/accountStatus';
+
+const CourierSchema = new Schema(
+  {
+    userId: { type: ObjectId, ref: 'User', required: true },
+    avatar: {
```

Comment:
this field is present in user schema, do we need to duplicate it here?

Reply:
It's optional to have different avatars for different roles

Reply:
yes, saw it in a thread for a different PR, ok